src/main/scala/com/cloudera/sparkts/TimeSeries.scala

```diff
@@ -169,6 +169,55 @@ class TimeSeries[K](val index: DateTimeIndex, val data: DenseMatrix,
   }
 
   /**
+    * Returns a TimeSeries where each individual time series is differenced with the given time
+    * frequency.
```

Comment:
I'm finding it somewhat difficult to reason about how this behaves.  Is the documentation able to give an example?  E.g. if the frequency is ten hours and the index being operated on has timestamps at noon, 1 pm, 2 pm, 10 pm, and 11 pm, what would be the output?